### ISSUE TITLE
Annotations is maximum 100?

### ISSUE THREAD

**Issue by 1902606-JeronePohYongCheng** [alankbi/detecto#88]
**Describe the bug**
While using Detecto library, is it always giving out a maximum of 100 annotations that is detected when training a custom object?

**Code and Data**
Share the code that caused the error as well as an example image/label from your dataset.

**Environment:**
 - OS: 
 - Python version: 
 - Detecto version:
 - torch version: 
 - torchvision version: 

**Additional context**

**Comment by alankbi** [comment]
It looks like this might be the case by default with PyTorch's models. I'd recommend taking a look at [this thread](https://discuss.pytorch.org/t/only-100-predictions-outputs-for-torchvision-object-detection-finetuning-tutorial/98029) and seeing if you can manually override that option with your model (you can access the underlying torchvision model using the `core.Model.get_internal_model` method)
